File: externals/openmpi-1.6.5/ompi/contrib/vt/vt/tools/opari/lib/pomp_test.f90
       program test2
       integer i

       !$pomp inst init

       write(*,*) "+++ sequential1"

       !$pomp inst begin(phase1)

       !$omp parallel do
       do i=1,4
         write(*,*) "+++ pdo",i
       enddo
       !$omp end parallel do

       !$pomp inst end(phase1)

       write(*,*) "+++ sequential2"

       !$omp parallelsections
         !$omp section
         write(*,*) "+++ psection 1"
         !$omp section
         write(*,*) "+++ psection 2"
       !$omp end parallelsections

       write(*,*) "+++ sequential3"

       end

File: externals/openmpi-1.6.5/ompi/contrib/vt/vt/tools/opari/lib/pomp_test.c
#include <stdio.h>

int main() {
  int i;

  #pragma pomp inst init

  printf("+++ sequential1\n");

  #pragma pomp inst begin(phase1)

    #pragma omp parallel for
    for (i=0; i<4; ++i) { printf("+++ pdo %d\n", i); }

  #pragma pomp inst end(phase1)

  printf("+++ sequential2\n");

  #pragma omp parallelsections
  {
    #pragma omp section
    printf("+++ psection 1\n");
    #pragma omp section
    printf("+++ psection 2\n");
  }

  printf("+++ sequential3\n");
  return 0;
}
